Data/ByteString/Builder/Internal.hs

```diff
@@ -84,6 +84,8 @@ module Data.ByteString.Builder.Internal (
   -- , sizedChunksInsert
 
   , byteStringCopy
+  , asciiLiteralCopy
+  , modUtf8LitCopy
```

Comment:
```suggestion
  , modUtf8LiteralCopy
```

For consistency with `asciiLiteralCopy` (or we might as well chose to use `Lit` for both)